src/modules/fancyzones/FancyZonesLib/FancyZonesLib.vcxproj

```diff
@@ -148,7 +148,9 @@
     </ProjectReference>
     <ResourceCompile Include="Generated Files/fancyzones.rc" />
     <None Include="packages.config" />
-    <None Include="Resources.resx" />
+    <None Include="Resources.resx">
+      <SubType>Designer</SubType>
```

Comment:
This change looks like an automatic change that's not part of the PR's goal. Can you please revert it?

Reply:
Done